docs/migration/6_0/html.md

```diff
@@ -206,7 +206,10 @@
 
 ### AddToCartComponent
 
-- Changed line 31 classes from `'link cx-action-link'` to `'btn btn-tertiary'`
+- Changed line 33 classes from `'link cx-action-link'` to `'btn btn-tertiary'`
+- Changed line 22 by wrapping `ng-template` in an `ng-container`
```

Comment:
I think it is not breaking change. adding `ng-container` has no effect.

Reply:
I will remove that line. Thanks